CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```

Comment:
I don't think we have to fully write out what everyone means,  
If you need to write out what everyone means i think there has gone something wrong. **Everyone** means **Everyone** to me.  

```suggestion
In the interest of fostering an open and welcoming environment, we as
contributors and maintainers pledge to making participation in our project and
our community a harassment-free experience for everyone.
```

Reply:
This was taken from [here](https://www.contributor-covenant.org/) as it is an easy drop-in. I do think that your suggestion is definitely much more understandable. Especially for people new to open-source development.